src/current/v24.3/disaster-recovery-overview.md

```diff
@@ -4,169 +4,23 @@ summary: Learn how to make your cluster resilient with high availability and dis
-    </td>
-    <td>1</td>
-    <td>3</td>
-  </tr>
-
-  <tr>
-    <td class="comparison-chart__feature">
-      <b>Fault tolerance</b>
-    </td>
-    <td>Zero RPO node, availability zone failures</td>
-    <td>Zero RPO node, availability zone failures, region failures</td>
-  </tr>
-
-</table>
-
-For details on designing your cluster topology for high availability with replication, refer to the [Disaster Recovery Planning]({% link {{ page.version.version }}/disaster-recovery-planning.md %}#hardware-failure) page.
-
-### Choose a disaster recovery strategy
-
-CockroachDB is designed to recover automatically; however, building [backups]({% link {{ page.version.version }}/backup-and-restore-overview.md %}) or [physical cluster replication]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}) into your disaster recovery planning is an important part of a resilient deployment.
-
-<table class="comparison-chart">
-  <tr>
-    <th></th>
-    <th>Point in time backup & restore</th>
-    <th>Physical cluster replication (asynchronous)</th>
-  </tr>
-
-  <tr>
-    <td class="comparison-chart__feature">
-      <b>RPO</b>
-    </td>
-    <td>>=5 minutes</td>
-    <td>10s of seconds</a></td>
-  </tr>
-
-  <tr>
-    <td class="comparison-chart__feature">
-      <b>RTO</b>
-    </td>
-    <td>Minutes to hours, depending on data size and number of nodes</td>
-    <td>Seconds to minutes, depending on cluster size</td>
-  </tr>
-
-  <tr>
-    <td class="comparison-chart__feature">
-      <b>Write latency</b>
-    </td>
-    <td>No impact</td>
-    <td>No impact</td>
-  </tr>
-
-  <tr>
-    <td class="comparison-chart__feature">
-      <b>Recovery</b>
-    </td>
-    <td>Manual restore</td>
-    <td>Manual failover</td>
-  </tr>
-
-  <tr>
-    <td class="comparison-chart__feature">
-      <b>Hardware cost</b>
-    </td>
-    <td>1x + S3-like storage & networking</td>
-    <td>2x + networking between datacenters</td>
-  </tr>
+{{site.data.alerts.callout_info}}
+For an overview of resiliency features in CockroachDB, refer to [Data Resilience]({% link {{ page.version.version }}/data-resilience.md %}).
+{{site.data.alerts.end}}
 
-  <tr>
-    <td class="comparison-chart__feature">
-      <b>Minimum regions</b>
-    </td>
-    <td>1</td>
-    <td>2</td>
-  </tr>
+## Choose a disaster recovery strategy
 
-  <tr>
-    <td class="comparison-chart__feature">
-      <b>Fault tolerance</b>
-    </td>
-    <td>Not applicable</td>
-    <td>Zero RPO node, availability zone, region failure with loss up to RPO</td>
-  </tr>
+CockroachDB is designed to recover automatically; however, building [backups]({% link {{ page.version.version }}/backup-and-restore-overview.md %}) or [physical cluster replication]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}) into your disaster recovery planning protects against unforeseen incidents.
 
-</table>
+{% include {{page.version.version}}/resilience/dr-feature-table.md %}
```

Comment:
docs nerd comment: this page should have a 'See also' section at bottom which links to the important things mentioned in the body 

(maybe premature at this level of drafting but figure i'll drop the comment)

Reply:
If possible, I think this would be a good add.

Reply:
Added to both pages a See also. Definitely a good point, I forgot this!